x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList/index.tsx

```diff
@@ -41,17 +50,39 @@ const AppLink = styled(TransactionOverviewLink)`
   ${truncate('100%')};
 `;
 
-export const SERVICE_COLUMNS = [
+export const SERVICE_COLUMNS: Array<ITableColumn<ServiceListItem>> = [
+  {
+    field: 'severity',
+    name: i18n.translate('xpack.apm.servicesTable.healthColumnLabel', {
+      defaultMessage: 'Health',
+    }),
+    width: px(unit * 6),
+    sortable: true,
+    render: (_, { severity }) => {
+      return <HealthBadge severity={severity} />;
+    },
+  },
   {
     field: 'serviceName',
     name: i18n.translate('xpack.apm.servicesTable.nameColumnLabel', {
       defaultMessage: 'Name',
     }),
     width: '40%',
     sortable: true,
-    render: (serviceName: string) => (
+    render: (_, { serviceName, agentName }) => (
       <EuiToolTip content={formatString(serviceName)} id="service-name-tooltip">
```

Comment:
```suggestion
      <EuiToolTip delay="long" content={formatString(serviceName)} id="service-name-tooltip">
```

Reply:
I would like to delay the tooltip on the service name (since we're showing it always for truncation). It's always been a pet peeve of mine and the way we're truncating these values in our tables.